apps/portals/e2e/explore.spec.ts

```diff
@@ -185,15 +185,17 @@ const toggleShowHideVisualizationsAndVerify = async (
   const showHideButton = page.getByRole('button', {
     name: 'Show / Hide Visualizations',
   })
-  const ageDistChart = page.getByText('Age Distribution')
-  const isAgeDistributionChartPresent = (await ageDistChart.count()) > 0
+  const chartsPanel = page.locator('.PlotsContainer')
 
-  const chartsPanel = page.locator('.FacetNav')
+  //const ageDistChart = page.getByText('Age Distribution')
+  const ageDistChart = chartsPanel.locator('.SynapsePlots:visible')
+  const isAgeDistributionChartPresent = (await ageDistChart.count()) > 0
```

Comment:
If possible, try to replace locators with roles.

Delete commented code once it's all working